src/main/webapp/resources/js/components/Buttons/ConfirmButton.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
```

Comment:
File comment and when to use this

Reply:
Updated in bbe476aac3c33275094afab6d2be49b9fd570d15